libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -262,6 +262,14 @@ class MapboxNavigation(
         tripSession.stop()
     }
 
+    /**
+     * Clears the location and route for the navigator.
+     */
+    fun resetTripSession() {
```

Comment:
Can we add a couple of tests around this new API?

Reply:
Test are failing in CI https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/9325/workflows/ab41e4ad-7d22-4571-b82b-ce0f136001ec/jobs/34959

```
> Task :libnavigation-core:compileDebugUnitTestKotlin FAILED
e: /root/code/libnavigation-core/src/test/java/com/******/navigation/core/MapboxNavigationTest.kt: (504, 53): Unresolved reference: Collections
```